assets/i18n/it.po

```diff
@@ -0,0 +1,779 @@
+#: assets/page/Help.svelte:38
+msgid "\"#\" will show the matching conversation names."
+msgstr "\"#\" mostrerà i nomi delle conversazioni corrispondenti."
+
+#: assets/page/Help.svelte:37
+msgid "\"/\" will show the available commands."
+msgstr "\"/\" mostrerà i comandi disponibili."
+
+#: assets/page/Help.svelte:36
+msgid "\":\" followed by a character will show matching emojis."
+msgstr "\":\" seguito da un carattere che mostrerà le emojis corrispondenti."
+
+#: assets/page/Help.svelte:35
+msgid ""
+"\"@\" followed by a character will show matching nicks in the current "
+"conversation."
+msgstr ""
+"\"@\" seguito da un carattere mostrerà i nick corrispondenti nella "
+"conversazione in corso."

```

Comment:
Unfortunately my “dumb” in-house .po parser doesn’t support multiple lines. Can you edit the file and put all strings on one line?